torch/_tensor_docs.py

```diff
@@ -6362,6 +6374,23 @@ def callable(a, b) -> number
 masked_scatter(mask, tensor) -> Tensor
 
 Out-of-place version of :meth:`torch.Tensor.masked_scatter_`
+
+.. note::
+
+    The :attr:`self` tensor and the :attr:`mask` tensor must be
+    :ref:`broadcastable <broadcasting-semantics>`. The resulting
+    :attr:`self` tensor will have, for each dimension, the maximum of
+    the sizes of :attr:`self` and :attr:`mask` along that dimension.
```

Comment:
What about simply the more concise
```suggestion
The inputs `self` and `mask` broadcast.
```
perhaps with a link to the broadcasting semantics?

Reply:
makes sense!